nuxeo-features/nuxeo-admin-center/nuxeo-admin-center-core/src/main/java/org/nuxeo/connect/client/we/PackageListingProvider.java

```diff
@@ -28,8 +28,7 @@
 import javax.ws.rs.Produces;
 import javax.ws.rs.QueryParam;
 
-import org.apache.commons.lang.StringUtils;
-
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Both are part of a "dependency" cleanup; as `commons.lang` was a used but not declared dependency. I found it better to use `commons.lang3` and align code instead of adding a dependency to the old one.